client/settings/express-checkout-settings/payment-request-button-preview.js

```diff
@@ -67,6 +69,42 @@ const buttonSizeToPxMap = {
 	large: 56,
 };
 
+const WooPayButtonPreview = ( { size, buttonType, theme } ) => (
+	<WoopayExpressCheckoutButton
+		isPreview={ true }
+		buttonSettings={ {
+			type: buttonType,
+			text: 'Buy',
+			theme: theme,
+			height: `${
+				buttonSizeToPxMap[ size ] || buttonSizeToPxMap.medium
+			}px`,
+			size,
+		} }
+	/>
+);
+
+const ButtonPreviewWrapper = ( { theme, children } ) => (
+	<>
+		<div className="payment-method-settings__preview" data-theme={ theme }>
+			{ children }
+		</div>
+		<BrowserHelpText />
+	</>
+);
+
+const PreviewRequirementsNotice = () => (
+	<InlineNotice icon status="info" isDismissible={ false }>
+		{ __(
+			'To preview the express checkout buttons, ' +
+				'ensure your store is served over HTTPS on a domain available to the public internet, ' +
+				'your device is configured to use Apple Pay or Google Pay, ' +
+				'and view this page using the Safari or Chrome browsers.',
+			'woocommerce-payments'
+		) }
```

Comment:
@elizaan36 @nikkivias @pierorocca In addition to the questions in the PR description, also looking for input on the info notice here. I think it's quite verbose but couldn't find any better, simpler, or more succinct phrasing that gets the point across.

Maybe we should add this to the documentation on PRBs and link to that instead?

Reply:
hey @reykjalin You can probably use the full text but maybe break it into two sentences to make it easier to read. 

Here's an option:
> To preview the express checkout buttons, ensure your store uses HTTPS on a publicly available domain and you're viewing this page on a Safari or Chrome browser. Your device must be configured to use Apple Pay or Google Pay.

Reply:
Updated in 4863d3315de40a437190029d5b561d50ed2ff30a.